modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -154,8 +153,36 @@ use {{invokerPackage}}\ObjectSerializer;
                     $response->getBody()
                 );
             }
+        {{#responses}}
+            {{#-first}}
+
+            $responseBody = $response->getBody();
+            switch($statusCode) {
+            {{/-first}}
+            {{#dataType}}
+                {{^isWildcard}}case {{code}}:{{/isWildcard}}{{#isWildcard}}default:{{/isWildcard}}
+                    if ('{{dataType}}' === '\SplFileObject') {
+                        $content = $responseBody; //stream goes to serializer
+                    } else {
+                        $content = $responseBody->getContents();
+                        if ('{{dataType}}' !== 'string') {
+                            $content = json_decode($content);
+                        }
+                    }
+
+                    return [
+                        ObjectSerializer::deserialize($content, '{{dataType}}', []),
+                        $response->getStatusCode(),
+                        $response->getHeaders()
+                    ];
+            {{/dataType}}
+            {{#-last}}
+            }
+            {{/-last}}
+        {{/responses}}
 
             {{#returnType}}
+			$returnType = '{{returnType}}';
```

Comment:
Please use space instead of tab for indention.

Reply:
Fixed